src/Web.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+module Web
+    ( webMain
+    , inMemoryApplication
+    )
+where
+
+import Control.Monad.RWS hiding (get)
+import Network.Wai.Middleware.Static
+import Network.Wai
+import Web.Spock.Safe
+
+import Kis
+import Kis.SqlBackend
+
+__ASSET_DIR__ :: FilePath
+__ASSET_DIR__ = "frontend"
+
+webMain :: IO ()
+webMain = runSpock 8080 inMemoryWeb
+
+web :: MonadIO m => (forall a. KisClient m a -> IO a) -> IO Middleware
+web runKisClient =
+    spockT runKisClient webInterface
+
+inMemoryWeb :: IO Middleware
+inMemoryWeb = do
+    backend <- inMemoryBackend
+    web (runClient backend)
+
```

Comment:
Ja, das ist noch ein bisschen ungünstig. Das Problem ist, dass `withInMemoryKis` bei jedem Aufruf erneut eine Datenbank-Connection erstellt (und z.B. auch die Migrations ausführt), während wir immer die gleiche Connection verwenden wollen. Also `spockT` ruft z.B. für jedes `req' (CreatePatient patient)` `runKisClient` auf. Wenn ich statt `runKisClient` `withInMemoryKis` übergebe, gibt's bei jedem Request eine neue leere Datenbank.
Sollte aber wenigstens irgendetwas a la `withInMemoryBackend` oder `withInMemoryPool` sein, damit die Connection in jedem Fall auch wieder geschlossen wird.